src/Tests/Microsoft.Diagnostics.Monitoring.Tool.TestHostingStartup/BuildOutputSharedLibraryInitializer.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Tools.Monitor;
+using Microsoft.Diagnostics.Tools.Monitor.Profiler;
+using Microsoft.Extensions.FileProviders;
+using Microsoft.Extensions.Logging;
+using System.IO;
+using System.Reflection;
+
+namespace Microsoft.Diagnostics.Monitoring.Tool.TestHostingStartup
+{
+    internal sealed class BuildOutputSharedLibraryInitializer : ISharedLibraryInitializer
+    {
+        private static readonly string SharedLibrarySourcePath =
+            Path.GetFullPath(Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "..", "..", ".."));
```

Comment:
Does this end up being the same for VS F5 scenarios and command line scenarios that call build.ps1?

Reply:
Yes, it's the same regardless of it being built by VS or command line.